adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -495,34 +540,49 @@ public AuthenticationResult acquireTokenSilentSync(String resource, String clien
             throw new IllegalArgumentException("The required clientId is null or blank.");
         }
 
+        final String requestId = Telemetry.registerNewRequest();
+        final APIEvent apiEvent = createApiEvent(mContext, clientId, requestId, EventStrings.ACQUIRE_TOKEN_SILENT_SYNC);
+        apiEvent.setSilentRequestPromptBehavior(PromptBehavior.Auto.toString());
```

Comment:
>apiEvent.setSilentRequestPromptBehavior(PromptBehavior.Auto.toString()); [](start = 8, length = 72)

Silent request shouldn't have a prompt behavior, isn't this causing confusion if you have a auto+silent?